src/applications/ivc-champva/10-10D/containers/ConfirmationPage.jsx

```diff
@@ -125,27 +125,7 @@ export function ConfirmationPage(props) {
   );
 }
 
-ConfirmationPage.propTypes = {
-  form: PropTypes.shape({
-    pages: PropTypes.object,
-    data: PropTypes.shape({
-      applicants: PropTypes.array,
-      statementOfTruthSignature: PropTypes.string,
-      veteransFullName: {
-        first: PropTypes.string,
-        middle: PropTypes.string,
-        last: PropTypes.string,
-        suffix: PropTypes.string,
-      },
-    }),
-    formId: PropTypes.string,
-    submission: PropTypes.shape({
-      response: PropTypes.shape({ confirmationNumber: PropTypes.string }),
-      timestamp: PropTypes.string,
-    }),
-  }),
-  name: PropTypes.string,
-};
```

Comment:
Pulled this out to a shared constants file because we have identical proptypes in other locations that can share this.